components/formats-gpl/src/loci/formats/in/ZeissCZIReader.java

```diff
@@ -88,6 +89,11 @@
 
   // -- Constants --
 
+  private static final String ALLOW_AUTOSTITCHING_KEY = "zeissczi.allow.autostitch";
```

Comment:
`classname.key` is OK, though I think I'd prefer sometime more general.  Class names are internal implementation details.  For example, I'd prefer `tiff` over `TiffWriter`.  It's usable by both readers and writers, and also compatible with implementations in other languages which may have different class names.  And it precludes changing the implementation without breaking the option names.  And for an option like e.g. `jpeg.quality`, it can be used by the TIFF writer when using JPEG compression.  So I think it's more user-friendly to use accessible prefixes which are simpler, more flexible, and don't expose private implementation details.